tools/debian/control

```diff
@@ -0,0 +1,116 @@
+Source: cockpit
+Section: admin
+Priority: optional
+Maintainer: Michael Biebl <biebl@debian.org>
```

Comment:
I have changed this to use 'Cockpit <cockpit@cockpit-project.org>'.  This isn't a real email address, though.